tests/RequirementChecker/RequirementsDumperTest.php

```diff
@@ -113,7 +113,7 @@ public function provideJsonAndLockContents(): Generator
                     [
                         'name' => 'acme/foo',
                         'require' => [
-                            'php' => '^7.3',
+                            'php' => '^7.4',
```

Comment:
are those changes really necessary?

Reply:
No, I guess not. 👍🏻 Just was thinking that test was for the lowest PHP version that's supported.

Reply:
Do you want me to revert the changes to this file?

Reply:
No it's fine. I was just confused by it initally